migrations/20200420073606_AddColumnsToIndexables.php

```diff
@@ -0,0 +1,66 @@
+<?php
+/**
+ * Yoast SEO Plugin File.
+ *
+ * @package WPSEO\Migrations
+ */
+
+use Yoast\WP\SEO\ORM\Yoast_Model;
+use YoastSEO_Vendor\Ruckusing_Migration_Base;
+
+/**
+ * Class AddColumnsToIndexables.
+ */
+class AddColumnsToIndexables extends Ruckusing_Migration_Base {
+
+	/**
+	 * Migration up.
+	 */
+	public function up() {
+		$tables = $this->get_tables();
+
+		$blog_id = \get_current_blog_id();
+		foreach ( $tables as $table ) {
+			$this->add_column( $table, 'blog_id', 'biginteger', [
+				'null'  => false,
+				'limit' => 20,
+			] );
+
+			$this->query( 'UPDATE ' . $table . ' SET blog_id = ' . $blog_id );
+		}
+
+		$this->add_column( $tables[0], 'language', 'string', [ 'null' => true, 'limit' => 32 ] );
```

Comment:
Would not use `$tables[0]` here, instead `$indexables_table = Yoast_Model::get_table_name( 'Indexable' );` and use that. Makes it much more obvious what's going on.